release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -586,6 +586,15 @@ You can decrease power consumption through the Node Tuning Operator by specifyin
 [id="ocp-4-11-auth"]
 === Authentication and authorization
 
+[id="ocp-4-11-post-installation"]
+=== Post-installation configuration
+
+[id="ocp-4-11-cluster-capabilities"]
+==== Cluster capabilities
+As a cluster administrator, you can enable cluster capabilities to select or deselect one or more optional components prior to installation or post installation.
```

Comment:
Suggest replacing 'prior to' with 'before'. IBM Style guide recommends the following:
 "prior to" (preposition) Do not use. Use "before". (page 336).

Reply:
fixed.